### PR TITLE
[quant][docs] Add docstrings for operators defined in torch.ops.quantized_decomposed namespace

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #89547

Summary:
no functionality changes

Test Plan:
NA

Reviewers:

Subscribers:

Tasks:

Tags:

cc @jianyuh @raghuramank100 @jamesr66a @vkuzo @jgong5 @Xia-Weiwen @leslie-fang-intel

### CODE REVIEW

File: torch/ao/quantization/fx/_decomposed.py [hunk 1]
```diff
@@ -32,6 +32,20 @@ def _quant_min_max_bounds_check(quant_min, quant_max, dtype):
 
 @impl(quantized_decomposed_lib, "quantize_per_tensor", "CompositeExplicitAutograd")
 def quantize_per_tensor(input, scale, zero_point, quant_min, quant_max, dtype):
+    """ Affine quantization for the Tensor using the same quantization parameters to map
```
Comment (vkuzo) [inline torch/ao/quantization/fx/_decomposed.py:35]: can we add types, both annotated in the code and also in the docstring?  For example, it's not clear if `scale` is a scalar or a tensor.

File: torch/ao/quantization/fx/_decomposed.py [hunk 2]
```diff
@@ -32,6 +32,20 @@ def _quant_min_max_bounds_check(quant_min, quant_max, dtype):
...
+    Args:
+       input: original float32 Tensor
+       scale: quantization parameter for affine quantization
+       zero_point: quantization parameter for affine quantization
+       quant_min: minimum quantized value for output Tensor
+       quant_max: maximum quantized value for output Tensor
+       dtype: requested dtype for output Tensor
+
+    Returns:
+       per tensor quantized Tensor with requested dtype
```
Comment (vkuzo) [inline torch/ao/quantization/fx/_decomposed.py:47]: is this true?  I thought this returns a tensor of integers, which need to be combined with scale and zero_point to interpret the values.  "per tensor quantized tensor" sounds like "tensor + scale + zero_point", which doesn't match the function

**Review by vkuzo — APPROVED** [review]

File: torch/ao/quantization/fx/_decomposed.py (hunk 2 above)
Comment (jerryzh168) [reply torch/ao/quantization/fx/_decomposed.py:47]: oh, makes sense, I'll explain this in a clearer way

**Pull request closed without merging**